doc/build/build.md

```diff
@@ -257,3 +257,11 @@ Add the following to the `Make.user` file:
     USE_INTEL_MKL = 1
 
 It is highly recommended to start with a fresh clone of the Julia repository.
+
+## Source distributions of releases
+
+Each pre-release and release of Julia has a "full" source distribution and a "light" source distribution.
+
+The full source distribution contains the source for Julia and all dependencies so that it can be built from source without an internet connection. The light source distribution does not include the dependencies.
```

Comment:
```suggestion
The full source distribution contains the source code for Julia and all dependencies so that it can be built from source without an internet connection. The light source distribution does not include the souce code of dependencies.
```